itest/lnd_multi-hop-payments_test.go

```diff
@@ -74,14 +74,29 @@ func testMultiHopPayments(ht *lntest.HarnessTest) {
 	const aliceFeeRatePPM = 100000
 	updateChannelPolicy(
 		ht, alice, chanPointAlice, aliceBaseFeeSat*1000,
-		aliceFeeRatePPM, chainreg.DefaultBitcoinTimeLockDelta,
+		aliceFeeRatePPM, 0, 0, chainreg.DefaultBitcoinTimeLockDelta,
 		maxHtlc, carol,
 	)
 
+	// Define a negative inbound fee for Alice, to verify that this is
+	// backwards compatible with an older sender ignoring the discount.
+	const (
+		aliceInboundBaseFeeMsat = -1
```

Comment:
That's correct. A routing node gets paid a single fee (the diff between in and out htlc amt). If this single fee is less than the required in+out fee, the routing node can only signal 'fee insufficient'. They can't know why the sender didn't include enough. It can be because of an outdated inbound fee policy, an outdated outbound fee policy, or both.

#6967 returns the inbound fee policy to the sender, so that they can figure out for themselves what they did wrong.